### PR TITLE
ZCS-13118 : Upgraded OpenLDAP to 2.5.17

### PR DESCRIPTION
**Pull request by umagmrit**

### CODE REVIEW

**Review by silentsakky — APPROVED** [review]

File: thirdparty/openldap/zimbra-openldap/debian/control [hunk 1]
```diff
@@ -12,7 +12,7 @@ Package: zimbra-openldap-server
 Priority: optional
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, zimbra-openldap-lib (= ${binary:Version}),
- zimbra-ldap-base
+ zimbra-ldap-base, zimbra-libsodium-dev (>= 1.0.19-1zimbra10.0b1ZAPPEND)
```
Comment (ghen2) [inline thirdparty/openldap/zimbra-openldap/debian/control]: You probably want `zimbra-libsodium-lib` here, instead of `-dev`.

**Review by ghen2 — APPROVED** [review]

File: thirdparty/openldap/zimbra-openldap/debian/control (hunk 1 above)
Comment (umagmrit) [reply thirdparty/openldap/zimbra-openldap/debian/control]: right, updated it.

**Review by dasiyogesh — APPROVED** [review]

File: thirdparty/openldap/zimbra-openldap/rpm/SPECS/openldap.spec [hunk 2]
```diff
@@ -1,20 +1,15 @@
...
 License:            BSD
 Source:             %{name}-%{version}.tgz
-Patch0:             ITS5037.patch
-Patch1:             writers.patch
-Patch2:             ITS7683.patch
-Patch3:             ITS8054.patch
-Patch4:             liblmdb-soname.patch
-Patch5:             multival.patch
-Patch6:             liblmdb-keysize.patch
-Patch7:             ITS9608.patch
```
Comment (ghen2) [inline thirdparty/openldap/zimbra-openldap/rpm/SPECS/openldap.spec:14]: Can you remove the dropped patches from the repo?

Comment (umagmrit) [reply thirdparty/openldap/zimbra-openldap/rpm/SPECS/openldap.spec:14]: Sure, will do that.

**Pull request merged**